src/main/java/it/gov/pagopa/receipt/pdf/notifier/service/impl/ReceiptToIOServiceImpl.java

```diff
@@ -55,6 +60,7 @@ public void notifyMessage(Map<String, UserNotifyStatus> usersToBeVerified,
                 (receipt.getIoMessageData() == null ||
                         ReceiptToIOUtils.verifyMessageIdIsNotPresent(userType, receipt)
                 )
+                && (!CF_FILTER_ENABLED || CF_FILTER_NOTIFIER.contains(fiscalCode))
```

Comment:
is it possible to use only `CF_FILTER_NOTIFIER` as configuration?
If it is empty, no message is notified, otherwise, only the enabled CFs are notified.
To enable all ( ex into DEV env ) we can use a special keyword `*` ?

Reply:
Shouldn't the allow all condition be the default for the notifier usage?